app/src/composables/use-item.ts

```diff
@@ -210,7 +210,7 @@ export function useItem<T extends Record<string, any>>(
 			if (relation.meta?.one_field && relation.meta.one_field in newItem) {
 				const fieldsToFetch = fields
 					.filter((field) => field.startsWith(relation.meta!.one_field!))
```

Comment:
I've not found any issue with this fix, but noticed an oddity with the use of `field.startsWith()` for filtering. 
Eg: `m2m` and `m2m_other` both start with `m2m`, resulting in duplicate fields requested. 

It might need to be updated to be more specific or it could be an issue with my local schema. 🤔 
Will have to investigate further, but it's probably out of the PR's scope.

Reply:
Fixed in 80c9893c6e5ce684747c10ac5cc85c50638eb368